### DISCUSSION TITLE
How to validate configs are taking effect?

### DISCUSSION THREAD

**Discussion by timbassler** [Digitalroot-Valheim/Digitalroot.Valheim.MaxDungeonRooms#11] [Q&A] [answered]
How can I validate that the mod is taking effect?

Valheim version 0.206.5
BepInEx version 5.4.1900
HookGenPatcher version 0.0.3
Jotunn version 2.5.1
Running via Thunderstore Mod Manager
Configured for 99/99 for testing purposes

I see the following during client startup:
`[Info   :   BepInEx] Loading [Digitalroot Max Dungeon Rooms 1.0.0]`

I start a new single player world, travel to the black forest, and see this as I approach a burial chamber:
```
[Info   : Unity Log] 02/18/2022 18:53:17: Loading dungeon

[Info   : Unity Log] 02/18/2022 18:53:17: Dungeon loaded 0

[Info   : Unity Log] 02/18/2022 18:53:17: Dungeon load time 1.9909 ms

[Info   : Unity Log] 02/18/2022 18:53:17: Available rooms:18

[Info   : Unity Log] 02/18/2022 18:53:17: To place:40

[Info   : Unity Log] 02/18/2022 18:53:17: Connections 4

[Info   : Unity Log] 02/18/2022 18:53:17: placed 10 doors

[Info   : Unity Log] 02/18/2022 18:53:17: Placed 30 rooms
```

I enter the dungeon and drop into freefly and see what looks like a small, vanilla dungeon (nowhere close to the 99/99 target that was configured):
![image](https://user-images.githubusercontent.com/7391069/154783470-89a0f932-9fbd-404a-acd2-6bde87293166.png)

I would think that the above `To place:40` should show `To place:99` per configuration.  Is this wrong? Is the dungeon simply running out of valid connection points early?  (And if so, is there any way to overcome this limitation?)

Any guidance is greatly appreciated!

**Comment by Digitalroot** [comment, marked as answer]
I suggest rereading the mod info page. TLDR, set to 1000. The 1000 is room placement attempts. Not that you will ever get 1000 rooms.

**Reply by timbassler** [reply]
Thanks for the quick reply!  I guess it's just random chance whether the dungeon turns out large like what you show in the screenshots or if it'll still be small like what I saw above.  I got the impression that the size difference would be pretty consistently noticeable, but I understand that the generation logic has its limitations.

**Reply by Digitalroot** [reply]
With the values set to 1000, and after finding a good number of dungeons. I hope it becomes clear that the mod is having an impact.
